cypress/e2e/advanced.cy.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types='cypress' />
+
+const doClick = require("../support/DoClick");
+
+describe('Book Store app', () => { 
+  const username = 'TestCypress';
+  const book = 'Speaking JavaScript';
+  const idBook= 'book=9781449365035';
+
+  beforeEach(() => {
+
+    cy.logInNewUser().as('user')
+    cy.visit('https://demoqa.com/profile') 
+  });
+
+  //afterEach(() =>{
+    //if (cy.contains((('#see-book-Speaking\ JavaScript > a')))) {
+       //doClick.click('#delete-record-undefined');
+       //doClick.click('#closeSmallModal-ok');
+    //}
```

Comment:
Example of the comment for 16-20 rows. See it, and close it, please.

Reply:
1. i deleted it, because i can user this afterEach only after 'Add book'